app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology.</p>
+<p>In an increasingly digital world, you could inspire pupils to look at a future role in technology. With roles in technology making up 14% of all job opportunities in the UK.</p>
```

Comment:
@caitlin-reidy With roles... isn't a complete sentence. Think it either needs to be: look at a future in tech, with roles in this sector making up ...

Or perhaps we could rewrite it to include a couple of examples of types of tech jobs e.g. the content below?

In an increasingly digital world, there's a growing need for people with tech skills. Whether it's understanding how to use artificial intelligence (AI) or creating apps, roles in tech make up 14% of job opportunities in the UK. And you can help pupils understand how tech can be part of their future career.